pahfit/base.py

```diff
@@ -311,7 +311,8 @@ def plot(self, axs, x, y, yerr, model, scalefac_resid=2):
             if isinstance(cmodel, BlackBody1D):
                 cont_components.append(cmodel)
                 # plot as we go
-                ax.plot(x, cmodel(x) * ext_model / x, "#FFB000", alpha=0.5)
+                # ax.plot(x, cmodel(x) * ext_model / x, "#FFB000", alpha=0.5)
+                ax.plot(x, cmodel(x) * ext_model / x, "limegreen", alpha=0.5)
```

Comment:
not clear why line #314 is uncommented and line #315 is added which seems identical aside from the color? 
BTW if you changes the plotting colors, the docs may need to be updated.

Reply:
Sorry this is an artifact from another branch I made based on this. Not sure why it made it here. I will fix this.